### PR TITLE
README: Link to company sites, not bugzilla bugs.

### PR DESCRIPTION
**Pull request by cpu**
I think linking to Bugzilla bugs was a good way to bootstrap the list when users weren't self-reporting, but as a link target from the README the bug comments are not very informative.

In the case where we aren't able to link to a specific page about linting (e.g. EJBCA docs) I think its best to link to top level company webpages. Links to bugzilla comments should be reserved for the PRs that add new entries.

### CODE REVIEW

**Comment by cpu** [comment]
Going to self-merge this to avoid bugging a reviewer for a small docs-only PR. Happy to revert/revise if feedback arrives from anywhere.

File: README.md [hunk 1]
```diff
@@ -197,19 +197,19 @@ Pre-issuance linting is **strongly recommended** by the [Mozilla root
...
+* [Sectigo](https://sectigo.com/) and [crt.sh](crt.sh)
+* [Digicert](https://www.digicert.com/)
 * [EJBCA](https://download.primekey.com/docs/EJBCA-Enterprise/6_11_1/adminguide.html#Post%20Processing%20Validators%20(Pre-Certificate%20or%20Certificate%20Validation))
-* Google Trust Services
-* [Government of Spain, FNMT](https://bugzilla.mozilla.org/show_bug.cgi?id=1495507#c8)
-* [Globalsign](https://cabforum.org/pipermail/public/2018-April/013233.html)
-* [GoDaddy](https://bugzilla.mozilla.org/show_bug.cgi?id=1462844#c6)
-* [Izenpe](https://bugzilla.mozilla.org/show_bug.cgi?id=1528290#c5)
+* [Google Trust Services](https://pki.goog/)
+* [Government of Spain, FNMT](http://www.fnmt.es/)
```
Comment (sleevi) [inline README.md]: https?

File: README.md [hunk 2]
```diff
@@ -197,19 +197,19 @@ Pre-issuance linting is **strongly recommended** by the [Mozilla root
 program](https://wiki.allizom.org/CA/Required_or_Recommended_Practices#Pre-Issuance_Linting).
 Here are some projects/CAs known to integrate with ZLint in some fashion:
 
-* [Camerfirma](https://bugzilla.mozilla.org/show_bug.cgi?id=1556806#c5)
-* [CFSSL](https://github.com/cloudflare/cfssl/pull/1015)
-* [Sectigo and crt.sh](https://groups.google.com/forum/#!msg/mozilla.dev.security.policy/sjXswrcsvrE/Nl3OLd4PAAAJ)
-* [Digicert](https://bugzilla.mozilla.org/show_bug.cgi?id=1550645#c9)
+* [Camerfirma](https://www.camerfirma.com/)
+* [CFSSL](https://github.com/cloudflare/cfssl)
+* [Sectigo](https://sectigo.com/) and [crt.sh](crt.sh)
```
Comment (sleevi) [inline README.md]: Do you need a scheme here for the crt.sh link?

File: README.md (hunk 1 above)
Comment (cpu) [reply README.md]: HTTPS hangs for me :man_shrugging:

File: README.md (hunk 2 above)
Comment (cpu) [reply README.md]: Oops, good catch, will fix. Thanks @sleevi!

Comment (cpu) [reply README.md]: https://github.com/zmap/zlint/pull/349

**Pull request merged**